shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/props/DataSourceProperties.java

```diff
@@ -119,4 +122,84 @@ public int hashCode() {
         }
         return Objects.hashCode(dataSourceClassName, stringBuilder.toString());
     }
+
+    /**
+     * can data source be aggregate.
```

Comment:
First letter of java doc should be upper case

Reply:
Fix in https://github.com/apache/shardingsphere/pull/16632